### PR TITLE
Show Institution label for institutions

### LINKED ISSUE #678: Differentiate galleries and institutions in the search results
Re: https://github.com/artsy/gravity/pull/10667

The gravity support was done and deployed. Going to use this small UI story to pair with @jonallured  and give some introduction of the repo.

- [ ] Institutions and Institutional Sellers should be displayed as Institution in the search result.

### PR DESCRIPTION
**Pull request by jonallured**
Closes #678

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please assign someone to your PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by jonallured** [comment]
Nudge @cavvia 😄

File: models/search_result.coffee [hunk 1]
```diff
@@ -50,7 +50,8 @@ module.exports = class SearchResult extends Backbone.Model
       else if @get('model') is 'partnershow'
         'show'
       else if @get('model') is 'profile'
-        'gallery'
+        isInstitution = _.contains(['PartnerInstitution', 'PartnerInstitutionalSeller'], @get('owner_type'))
+        if isInstitution then 'institution' else 'gallery'
```
Comment (cavvia) [inline models/search_result.coffee]: I wonder if native coffeescript would be cleaner here:

```js
if @get('owner_type') in ['PartnerInstitution', 'PartnerInstitutionalSeller'] then 'institution' else 'gallery'
```

**Comment by cavvia** [comment]
Sorry for the delay, this got buried in my notifications! The types `PartnerInstitution` and `PartnerInstitutionalSeller` look correct, and the search result template should display the assigned `display_model` by default, so this looks :+1:

I've left a minor comment on syntactic elegance, but it's not a blocker.

**Comment by jonallured** [comment]
Yes, elegance is what I was going for, haha! With 164de6b, I think I got there. The local for the types seems legit to me, but your tip makes the line read nicely.

**Comment by cavvia** [comment]
Looks good!

**Pull request merged**